src/Modal/AlertModal.jsx

```diff
@@ -39,7 +44,7 @@ AlertModal.propTypes = {
   /** Sizes determine the maximum width of the dialog box */
   size: PropTypes.oneOf(['sm', 'md', 'lg', 'xl', 'fullscreen']),
   /** The visual style of the dialog box */
-  variant: PropTypes.oneOf(['default', 'warning', 'danger', 'success', 'dark']),
+  variant: PropTypes.oneOf(['default', 'warning', 'danger', 'success']),
```

Comment:
Looks like `ModalDialog` still has logic for handling a "dark" variant
https://github.com/openedx/paragon/blob/77a0dccdf66778ba85980b1a254c8caa791dd507/src/Modal/ModalDialog.jsx#L54
I assume that's something we want to move away from, and changing that feels out of scope of this PR, but I figured it's worth making a note of/possibly making an issue for.

Reply:
@brian-smith-tcril This may be a good discussion topic for a future Paragon Working Group meeting to get clarity. Even though `ModalDialog` supports "dark" variant, `AlertModal` itself does not as spec'd so makes sense to remove for now!